src/cli.js

```diff
@@ -29,9 +29,20 @@ export async function search (queryString, options) {
       combineWith: options.strict ? 'AND' : 'OR'
     });
 
-    await sendOutput(res, options, queryString);
-
-    return res;
+    // format res before giving final output JSON
+    const formattedRes = res.map(article => ({
```

Comment:
This mapping fixes the data input and output schema, making the function less reusable (e.g. what if the input article schema changes such as fetching from another organizations). An alternative is to pass a formatting function.